### ISSUE TITLE
Not getting logs for snuffulupagus issues

### ISSUE THREAD

**Issue by micah** [nbs-system/snuffleupagus#173]
I added this rule:

```
sp.disable_function.function("function_exists").param("function_name").value_r("(?:eval|exec|system)").drop();
```

and reloaded php-fpm, and I found that I could no longer see lists of files in nextcloud. So I went to look in the logs to figure out what was causing it, but I found no logs from snuffulupagus indicating anything. I looked in the /var/log/php7.0-fpm.log, the journal, and everywhere I could to try to find something. 

This is a pretty basic debian stretch install, with very little configured thus far.

**Comment by jvoisin** [comment]
> 16:44 @micah > jvoisin: i found snuff errors in my nginx error.log!

\o/

**Comment by micah** [comment]
Yeah, I was not expecting them to be there... I would say that this
issue could be closed.
